public/js/mc-answer.js

```diff
@@ -6,7 +6,7 @@ var mcAnswerCount = 4; // default number of options
 var addMCAnswers = function(dom){
     mcAnswerCount++;
     var newdiv = document.createElement('p');
-    var inputdiv = '<input type="radio" name="radbutton" value="mcans{0}" id="mcans{1}" required/><label for="mcans{2}">Is Correct</label><input type="text" name="mcans{3}" placeholder="Enter Answer Here" required style="float:left;" class="form-control"/><a onclick="$(this).parent().remove()" class="btn-floating btn-tiny waves-effect waves-light red"><i class="tiny material-icons">close</i></a>';
-    newdiv.innerHTML = inputdiv.format(mcAnswerCount,mcAnswerCount,mcAnswerCount,mcAnswerCount);
+    var inputdiv = '<input type="radio" name="radbutton" value="mcans{0}" id="mcans{0}" required/><label for="mcans{0}">Is Correct</label><input type="text" name="mcans{0}" placeholder="Enter Answer Here" required style="float:left;" class="form-control"/><a onclick="$(this).parent().remove()" class="btn-floating btn-tiny waves-effect waves-light red"><i class="tiny material-icons">close</i></a>';
+    newdiv.innerHTML = inputdiv.format([mcAnswerCount]);
```

Comment:
When you are fixing your conflicts, remember to keep the develop branch changes.

Reply:
yeah no worries